### ISSUE TITLE
QMs Prioritizing Getting the Ruling Correct

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#22]
Add language that QMs do not have to only use language presented in challenges, rebuttals, protests, when making a ruling. They are allowed to make a ruling, on "new" grounds, even if those grounds weren't presented to them during a challenge, rebuttal, protest.

Also, clarify whether a QM can Overrule a Challenge AND Change their Ruling.

**Comment by scottpeterson** [comment]
I would also include the Rule Book label.

**Comment by kclimenhaga** [comment]
The rule book currently does not state whether officials must only take into account the material presented in the challenge; I do not see anything in the rule book that would prohibit table staff from overturning a ruling based on information not presented in the challenge. I agree that table staff should have this flexibility if needed, but I am not sure whether a change to the rule book is needed.

**Comment by gryphonshafer** [comment]
I agree with kclimenhag that logically there doesn't need to be an explicit rule, but I've seen some (wrongly) assume the absence of an explicit allows to mean a restriction. So I also agree with scottpeterson that there's probably some value to something existing. Thus, this feels to me more like the something should be a "best practice" or maybe a "commentary" section.

**Comment by jswingle** [comment]
I think it's good in a commentary section (I think that fits better than a best practice in this use case).

**Comment by jswingle** [comment]
Revisiting this issue - I like the idea of specifying the overruled challenge part. In my judgment, a challenge should only be considered "overruled" if it failed to change the officials' minds at all. In a scenario where a quizzer challenges "the quizzer should be counted incorrect because of X" and the quizmaster rules "we will redo the question because of X," I think the quizzer still succeeded at their challenge and was not overruled, even if their exact suggestion for redress wasn't taken.
